### PR TITLE
Simplify the function check_if_all_files_exist

### PR DESCRIPTION
**Pull request by Tankypon**
Just simplify this function. I think it would be better like that if you want to add newly required files ;)

### CODE REVIEW

**Comment by bharadwaj-raju** [comment]
Appears good, but if `time` is False, then there is no day and night, too. Just weather. Referring to lines `248-249`.

**Comment by Tankypon** [comment]
I'm in total agreement. I remove the last `else` in my factorization (when time is false) and move the declaration of `required_files` over the `if time:`

If `time` is False then the variable `required_files` is not modified because the last `else` you seen in my code refers to the second `if` the one with `if args.time == 3:` . 

Normally the code is well indented. I checked it this morning with passing the argument --time and I think it works. Or maybe I didn't understand well your answer. Let me know!

File: WeatherDesk.py [hunk 1]
```diff
@@ -232,38 +232,25 @@ def check_if_all_files_exist(time=False, level=3):
...
-            'day-snow', 'night-snow',
-            'day-thunder', 'night-thunder',
-            'day-wind', 'night-wind',
-            'day-cloudy', 'night-cloudy']
-
-    else:
+            daytime = ['day', 'night']
 
-        required_files = ['rain', 'snow', 'normal', 'cloudy', 'wind', 'thunder']
+        required_files = [moment + '-' + weather
```
Comment (Tankypon) [inline WeatherDesk.py]: This is just executed when `time` is not False

**Comment by bharadwaj-raju** [comment]
Looking great! Have you tested it with all options?

**Comment by bharadwaj-raju** [comment]
Great! It works!

Accepting.

**Pull request merged**